images/techdocs/context/Makefile

```diff
@@ -8,7 +8,7 @@ docker_compose := $(docker) compose
 docker_compose_run := $(docker_compose) run --rm -T
 
 BASE_BRANCH := origin/main
-MARKDOWN_FILES := $(shell git diff --name-only --diff-filter=ACMRT $(BASE_BRANCH) | grep .md$$)
+# MARKDOWN_FILES := $(shell git diff --name-only --diff-filter=ACMRT $(BASE_BRANCH) | grep .md$$)
```

Comment:
Should this be commented out? It looks like it is being used below in `lint-fix`.

Reply:
nice catch